### ISSUE TITLE
Fehlende oder fehlerhafte Termine

### ISSUE THREAD

**Issue by wrkb** [RRZE-Webteam/fau-events#3]
...Problem ist mündlich bekannt, hier Beiträge zur Analyse und Problemlösung bzw. neue Fälle.
Aktuell hilft bei "hängenden" Terminen nur das manuelle Leeren (?) in einer Tabelle via RRZE.

**Comment by wrkb** [comment]
Neuester Fall: Aus einem Termin mit einen Link im Backend (Exchange)...

![unbenannt](https://cloud.githubusercontent.com/assets/9445370/6869062/600c089e-d491-11e4-8d87-aead5540cfe1.PNG)
...
wird ein Termin mit 2 Links im Frontend:

![unbenannt2](https://cloud.githubusercontent.com/assets/9445370/6869070/6f9ee8b2-d491-11e4-878a-b1e597faf4c1.PNG)

Hintergrund: Der Link im Termin wurde geändert, es waren aber nie zwei Links gleichzeitig drin. Ggf. kann ich noch Details zum geänderten Link einholen, wenn nötig.

**Comment by wrkb** [comment]
Konntet ihr den o.g. Fehler schon prüfen? Die Kollegin hat nachgefragt...
@rvdforst

**Comment by rvdforst** [comment]
Nein, das ist kein Fehler des iCal-Parsers. Der ursprünglicher Text zum Termin (Quelle) lautet:

Anmeldungen zum Karriere-Tag sind ab Mitte April 2015 über StudOn möglich: <http://www.studon.uni-erlangen.de/crs1043813.html> http://www.studon.uni-erlangen.de/crs1180313.html

Ich vermute, dass der Link durch "Copy-Paste" im Text eingefügt wurde. Also, der Link muss wieder eingegeben werden (kein Copy-Paste).
